Majasdarbs/02-majasdarbs/Portfolio/default_template.html

```diff
@@ -1,29 +1,105 @@
         <address>
-          <h1>Contact info:</h1>
+          <h1>My contact info:</h1>
+          <p>Phone number:+371 22011589</p>
```

Comment:
Link phone number https://www.elegantthemes.com/blog/wordpress/call-link-html-phone-number#:~:text=Adding%20an%20HTML%20Phone%20Number%20Call%20Link%20to%20your%20Website&text=Href%3Dtel%3A%20creates%20the%20call,the%20number%20it%20will%20call.